### PR TITLE
Create initial device module 

### PR DESCRIPTION
**Pull request by taymoork2**
# Pull Request Template

## Description

Create initial device module

Fixes [#34297](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-34297)

## Type of Change

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/devices/index.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+/*!
+ * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
+ */
+
+/**
+ *
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/devices/index.js]: Probably don't need these extra lines.

File: packages/node_modules/@webex/plugin-meetings/src/devices/index.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  requestPair(id, token) {
+    return new Promise((resolve, reject) => {
+      this.id = id;
+      this.pairedToken = token;
+      resolve(true);
+    });
+  }
+
+  /**
+   * TODO: Can a participant pair with a single device or multiple devices?
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/devices/index.js]: Please add a description too.

File: packages/node_modules/@webex/plugin-meetings/src/devices/index.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  /**
+   *
+   * Requires someone to be paired.
+   * @param {string} id
+   * @returns {Promise}
+   * @memberof Devices
+   */
+  join(id) {
+    return new Promise((resolve, reject) => {
+      if (!this.isPaired) reject();
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/devices/index.js]: I'm surprised the linter didn't pick this up. I would move this to two lines for consistency.

File: packages/node_modules/@webex/plugin-meetings/src/devices/index.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+/**
+ *
+ *
+ * @export
+ * @class Devices
+ */
+export default class Devices {
+  /**
+   * Create an instance of Devices.
+   * @param {*} {
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/devices/index.js]: Can we split these params out?

File: packages/node_modules/@webex/plugin-meetings/src/devices/index.js [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+   *
+   * Join meeting with device
+   * @param {string} id
+   * @returns {Promise}
+   * @memberof Devices
+   */
+  join(id) {
+    return new Promise((resolve, reject) => {
+      if (!this.isPaired) {
+        reject();
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/devices/index.js]: Should we reject with a `new Error(`? Would make the rejection easier to process

**Comment by taymoork2** [comment]
Incorrect module created
Created Device module instead of devices
Closing and creating a new PR

**Pull request closed without merging**